releasenotes/notes/adjust-symbolic-pulses-amp-angle-representation-f5c40007416cf938.yaml

```diff
@@ -0,0 +1,32 @@
+---
+other:
+  - |
+    Qiskit Terra 0.23.0, began phasing out support of complex parameters
+    in the Pulse module. Mainly, all library symbolic pulses were converted
+    from complex amplitude representation to a duo of real (float) parameters
+    (amp, angle). To avoid problems, Qiskit Experiments adopted this convention.
```

Comment:
Somewhere we should note that storing complex values for calibration parameters in general is now pending deprecation.

Reply:
Thank you.